ssz/sedes/byte_vector.py

```diff
@@ -59,6 +59,9 @@ def get_hash_tree_root(self, value: bytes) -> bytes:
         serialized_value = self.serialize(value)
         return merkleize(pack_bytes(serialized_value))
 
+    def chunk_count(self) -> int:
+        return self.length * self.size
+
```

Comment:
i'd suggest we delete this file and just define the `bytesN` (which iirc we do use in other places) in (e.g.) the `__init__.py` and just use the `Vector[Byte,N]` construction instead of keep this specialized implementation around.

Reply:
A `Vector[Byte, N]` would iterate over the single bytes for serialization/deserialization which is needlessly inefficient, that's why we have this optimized version. As far as I know we haven't benchmarked it though.

Reply:
I tried to replace it with `Vector(Byte, N)` from our codebase, it's also helpful for dump/parse as `bytes` instead of an array of `Byte()`. Keep the status quo here. :)

Reply:
gotcha thanks @jannikluhn -- i'd imagine we could detect if our `Vector[T,N]` has `T == byte` and do the optimizations behind the curtain but i'll also agree status quo works fine for now and may not be worth the maintenance overhead